src/diffusers/pipelines/stable_diffusion/pipeline_onnx_stable_diffusion.py

```diff
@@ -259,6 +259,7 @@ def __call__(
 class StableDiffusionOnnxPipeline(OnnxStableDiffusionPipeline):
     def __init__(
         self,
+        vae_encoder: OnnxRuntimeModel,
```

Comment:
Alt 1/2: remove this line

Reply:
Yes, the old pipeline didn't have an encoder, so let's remove and substitute it with **kwargs